start.js

```diff
@@ -4,34 +4,33 @@
 
 require('dotenv').config()
 
-const path = require('path')
-const fs = require('fs')
 const assert = require('assert')
 const updateNotifier = require('update-notifier')
 const PinoColada = require('pino-colada')
 const pump = require('pump')
-const resolveFrom = require('resolve-from')
 const isDocker = require('is-docker')
 const listenAddressDocker = '0.0.0.0'
 const watch = require('./lib/watch')
 const parseArgs = require('./args')
+const { exit, requireFastifyForModule, requireServerPluginFromPath, showHelpForCommand } = require('./util')
 
 let Fastify = null
 let fastifyPackageJSON = null
 
 function loadModules (opts) {
   try {
-    const basedir = path.resolve(process.cwd(), opts._[0])
+    const { module: fastifyModule, pkg: fastifyPkg } = requireFastifyForModule(opts._[0])
 
-    Fastify = require(resolveFrom.silent(basedir, 'fastify') || 'fastify')
-    fastifyPackageJSON = require(resolveFrom.silent(basedir, 'fastify/package.json') || 'fastify/package.json')
+    Fastify = fastifyModule
+    fastifyPackageJSON = fastifyPkg
   } catch (e) {
     module.exports.stop(e)
   }
 }
 
 function showHelp () {
```

Comment:
I think it is unnecessary to make `showHelpForCommand` inside `showHelp`. You can use `exit()` inside `showHelpForCommand`, and delete `showHelp` there.